plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Rules.hs

```diff
@@ -53,22 +56,44 @@ queueForEvaluation ide nfp = do
     EvaluatingVar var <- getIdeGlobalState ide
     modifyIORef var (Set.insert nfp)
 
-#if MIN_VERSION_ghc(9,0,0)
+#if MIN_VERSION_ghc(9,2,0)
+getAnnotations :: Development.IDE.GHC.Compat.Located HsModule -> [LEpaComment]
```

Comment:
maybe nest these two in a `where` clause of `apiAnnComments'` so that the same names are defined in all versions?

Reply:
I'll address that in a followup commit.